playground/nats/Nats.ApiService/Program.cs

```diff
@@ -0,0 +1,17 @@
+using NATS.Client.Core;
```

Comment:
Can we make this app a little more real? Can it use pubsub as well?

Reply:
sure, let me have a go!

Reply:
I created a simple JetStream example since it's easier demonstrate persisted messages using Web API. As for pubsub I suppose we'd need a frontend using SignalR or Blazor. Maybe a simple chat or something?